### PR TITLE
feat: add project README.md

### PR DESCRIPTION
**Pull request by afrodynamic**
# Pull Request Template

## Description

This PR adds a README file to the project to provide an overview of the application and how to setup and run it.

## Type of Change

- documentation: improvements or additions to documentation

## Checklist

- [x] I have read the [contributing guidelines](CONTRIBUTING.md) for this project.
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

## Reviewers
@lederius @ika-ika-ika

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Atelier
```
Comment (ika-ika-ika) [inline README.md:1]: Nice title!

Comment (afrodynamic) [reply README.md:1]: Thanks!

File: README.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+Atelier is a retail website project for Hack Reactor's "Front End Capstone".
+
+## Table of Contents
```
Comment (ika-ika-ika) [inline README.md:5]: Nice TOC

**Review by ika-ika-ika — APPROVED** [review]
Overall excellent work

**Review by lederius — CHANGES_REQUESTED** [review]
good

**Review by afrodynamic — COMMENTED** [review]
Looks good

**Review by lederius — APPROVED** [review]

**Comment by lederius** [comment]
finished

**Pull request merged**